app/internal/page/donor/how_should_replacement_attorneys_step_in.go

```diff
@@ -46,17 +47,13 @@ func HowShouldReplacementAttorneysStepIn(tmpl template.Template, lpaStore LpaSto
 					return err
 				}
 
-				redirectUrl := appData.Paths.TaskList
-
-				if len(lpa.Attorneys) > 1 &&
-					lpa.HowAttorneysMakeDecisions == page.JointlyAndSeverally &&
-					lpa.HowShouldReplacementAttorneysStepIn == page.AllCanNoLongerAct &&
-					len(lpa.ReplacementAttorneys) > 1 {
-					redirectUrl = appData.Paths.HowShouldReplacementAttorneysMakeDecisions
-
+				if len(lpa.Attorneys) > 1 && lpa.AttorneyDecisions.How == actor.JointlyAndSeverally && lpa.HowShouldReplacementAttorneysStepIn == page.AllCanNoLongerAct && len(lpa.ReplacementAttorneys) > 1 {
```

Comment:
I was going to suggest abstracting this into a func but what the hell would you call it? It's still readable enough with the 4 conditions I guess.